applications/openshift/networking/routes_rate_limit/tests/no_routes_rate_limit.pass.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
```

Comment:
why is this passing if no routes have rate limiting?

Reply:
Because we are checking non openshift namespaces, so a new cluster will pass the test